sambamba.wdl

```diff
@@ -20,13 +20,56 @@ version 1.0
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+task Flagstat {
```

Comment:
samtools flagstat uses a faster decompression library than sambamba, so it is usually faster.

BAM files are in BGZF (bgzip) format which is basically chunks of ~64KB compressed with the DEFLATE algorithm. Decompressing these chunks with zlib takes some time. Especially verifying the checksum with crc32. Samtools uses libdeflate internally that has optimized these  calculations.